cfme/fixtures/v2v_fixtures.py

```diff
@@ -29,6 +29,16 @@
 V2vProviders = namedtuple("V2vProviders", ["vmware_provider", "rhv_provider", "osp_provider"])
 
 
+def set_skip_event_history_flag(appliance):
+    """This flag is required for OSP to skip all old events and refresh inventory"""
+    appliance.update_advanced_settings(
+        {'ems': {'ems_openstack': {'event_handling': {'event_skip_history': True}}}})
+    appliance.server.logout()
```

Comment:
I don't think there is a need to perform server logout since the update is happening via REST. Is there a reason you're doing so?

Reply:
It worked without logout .. Removing it